### DISCUSSION TITLE
Upgrade

### DISCUSSION THREAD

**Discussion by vagdami** [abraunegg/onedrive#1422] [Q&A] [answered]
Hello,

I am using a this version (onedrive v2.4.2-4) but I would will upgrade for a new version (onedrive v2.4.11). Where can I get the guide for the upgrade?

Grateful
Vagner

**Comment by abraunegg** [comment, marked as answer]
@vagdami 
To upgrade your client you need to review the following document: https://github.com/abraunegg/onedrive/blob/master/docs/INSTALL.md

If you wish to keep building your own client, you need to first perform a `make uninstall` to uninstall the older binary, then proceed with building the new client and installing it.

If you wish to migrate to using a package for your distribution, use the instructions provided to install for your distribution.  Take note however the distribution versions are generally older and I do not support anything that is not the current release or 'master'.

To ensure that you keep up to date, 'star' and 'watch' this GitHub repository so that you are notified when new releases occur to allow you to rebuild your client with the latest version. Releases are typically monthly unless there is a critical issue or API change.
